src/components/QuestionForm/index.js

```diff
@@ -0,0 +1,21 @@
+import StyledQuestionForm from './styled';
+
+export default function QuestionForm({onSubmitFunction}) {
```

Comment:
I'd suggest calling the handler `onSubmit` (it is implied that this is a function)

Reply:
done